### PR TITLE
DCOS-53347: add error badges to Jobs form

### PR DESCRIPTION
**Pull request by mperrotti**
## Testing
Go to the jobs form, make some errors happen, confirm that the tabs get badges

For example:
1. Create a job with the id "test-id"
2. Select the "Command Only" radio input
3. Submit
4. The "General" tab should be badged with 1 error

## Trade-offs
The messages in the ErrorsAlert banner are treated differently than in the Create Service form:
- They aren't de-duplicated
- The error path is prepended to the error messages

Not de-duplicating the errors causes the error badge count not to match the number of list items in the ErrorsAlert banner

## Dependencies
[dcos-ui-plugins-private PR](https://github.com/mesosphere/dcos-ui-plugins-private/pull/1055)

## Screenshots
Screen Shot 2019-08-07 at 5 14 50 PM

### CODE REVIEW

**Comment by GeorgiSTodorov** [comment]
A lot of tests are failing, I will take a look.

File: plugins/services/src/js/service-configuration/ServiceConfigDisplay.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { MountService } from "foundation-ui";
 
 import ErrorsAlert from "#SRC/js/components/ErrorsAlert";
 import ConfigurationMap from "#SRC/js/components/ConfigurationMap";
-import { translateErrorMessages } from "#SRC/js/utils/ErrorMessageUtil";
+import ErrorMessageUtil from "#SRC/js/utils/ErrorMessageUtil";
```
Comment (GeorgiSTodorov) [inline plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: Why was this changed?

File: tests/pages/jobs/JobCreateForm-cy.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      jobDetails: true,
+      mesos: "1-for-each-health",
+      nodeHealth: true
+    });
+    cy.visitUrl({ url: "/jobs/overview" });
+  });
+
+  it("displays an error badge for the Services tab", function() {
+    const jobName = "simple";
+    const fullJobName = `${Cypress.env("TEST_UUID")}.${jobName}`;
```
Comment (GeorgiSTodorov) [inline tests/pages/jobs/JobCreateForm-cy.js]: This is only for system tests.

File: plugins/jobs/src/js/components/JobsForm.tsx [hunk 3]
```diff
@@ -300,19 +303,41 @@ class JobsForm extends React.Component<JobsFormProps, JobsFormState> {
...
+        <TabButton
+          id={item.id}
+          label={<Trans render="span" id={item.label} />}
+          key={item.key}
+          count={finalErrorCount}
+          showErrorBadge={Boolean(finalErrorCount) && this.props.showAllErrors}
+          description={
+            // TODO: pluralize
+            <Trans render="span">
+              {finalErrorCount} issues need addressing
```
Comment (GeorgiSTodorov) [inline plugins/jobs/src/js/components/JobsForm.tsx]: Can this be `issueCount`, since we already have a translation: https://github.com/dcos/dcos-ui/blob/235e9b80964f96fa7606dad1fb5f4269a33ce0c5/locale/zh/messages.json#L1008

**Comment by GeorgiSTodorov** [comment]
@mperrotti , I updated the integration tests, I hope you don't mind.

File: plugins/services/src/js/service-configuration/ServiceConfigDisplay.js (hunk 1 above)
Comment (mperrotti) [reply plugins/services/src/js/service-configuration/ServiceConfigDisplay.js]: For some reason, Typescript doesn't like destructured imports. It errors with "`translateErrorMessages` is not a function" when I do a destructured import

File: tests/pages/jobs/JobCreateForm-cy.js (hunk 2 above)
Comment (mperrotti) [reply tests/pages/jobs/JobCreateForm-cy.js]: We use `Cypress.env("TEST_UUID")}` in other jobs integration tests, but I'll drop it here
